src/Symfony/Bundle/SecurityBundle/DependencyInjection/Compiler/ReplaceDecoratedRememberMeHandlerPass.php

```diff
@@ -37,9 +37,6 @@ public function process(ContainerBuilder $container): void
 
             // get the actual custom remember me handler definition (passed to the decorator)
             $realRememberMeHandler = $container->findDefinition((string) $definition->getArgument(0));
-            if (null === $realRememberMeHandler) {
```

Comment:
Why can this be removed?

Reply:
`$container->findDefinition()` return type is non-nullable `Definition`